keyboards/smart/smart80/smart80.c

```diff
@@ -0,0 +1,350 @@
+/* Copyright 2022 Jacky
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+ 
+#include "smart80.h"
+#include "uart.h"
+//#include "bootloader.h"
+extern uint8_t keyboard_led_state;
+
+
+void board_init(void) {
+    uart_init(115200);
+    uart_write('s');
+}
+
+__attribute__((weak)) void bootloader_jump(void) {
+    BKP->DR1 = 0x4F42;
+    NVIC_SystemReset();
+}
+
+__attribute__((weak)) void mcu_reset(void) {}
+
+__attribute__((weak)) void enter_bootloader_mode_if_requested(void) {}
```

Comment:
```suggestion
void bootloader_jump(void) {
    BKP->DR1 = 0x4F42;
    NVIC_SystemReset();
}
```

Also, wouldn't you need this in the smart70 board, too?